connect/runtime/src/main/java/org/apache/kafka/connect/runtime/rest/util/SSLUtils.java

```diff
@@ -114,7 +114,7 @@ protected static void configureSslContextFactoryAlgorithms(SslContextFactory ssl
         if (sslCipherSuites != null)
             ssl.setIncludeCipherSuites(sslCipherSuites.toArray(new String[sslCipherSuites.size()]));
 
-        ssl.setSslKeyManagerFactoryAlgorithm((String) getOrDefault(sslConfigValues, SslConfigs.SSL_KEYMANAGER_ALGORITHM_CONFIG, SslConfigs.DEFAULT_SSL_KEYMANGER_ALGORITHM));
+        ssl.setKeyManagerFactoryAlgorithm((String) getOrDefault(sslConfigValues, SslConfigs.SSL_KEYMANAGER_ALGORITHM_CONFIG, SslConfigs.DEFAULT_SSL_KEYMANGER_ALGORITHM));
```

Comment:
cc @rhauch

Reply:
This change is good, since it's recommended as part of the original method's deprecation documentation.